### DISCUSSION TITLE
EMQ X 4.3.8 Released

### DISCUSSION THREAD

**Discussion by Rory-Z** [emqx/emqx#5659] [General]
EMQ X 4.3.8 is released now, it mainly includes the following changes:

**Bug fixes:**

- Fix the rule engine rule import fails

  Github PR: [emqx#5512](https://github.com/emqx/emqx/pull/5512)

- Fix the Path field in the Webhook action of rule engine cannot be used

  Github PR: [emqx#5468](https://github.com/emqx/emqx/pull/5468)

- Fix the issue that the Force Shutdown mechanism cannot take effect when the process is suspended

  Github PR: [emqx#5460](https://github.com/emqx/emqx/pull/5460)

- Fix the issue that the k8s deployment EMQ X cluster cannot be restarted correctly in some cases

  Github PR: [emqx#5646](https://github.com/emqx/emqx/pull/5646), [emqx#5428](https://github.com/emqx/emqx/pull/5428)

- Fix exproto cross-node process call error

  Github PR: [emqx#5436](https://github.com/emqx/emqx/pull/5436)

**Enhancement:**

- Add automatic reconnection mechanism and related configuration items for request timeout for exhook to enhance reliability

  Github PR: [emqx#5447](https://github.com/emqx/emqx/pull/5447)

- Add disconnect retry mechanism for exproto

  Github PR: [emqx#5436](https://github.com/emqx/emqx/pull/5436)

> Note: Starting from this version, CentoOS 7 requires the use of openssl 1.1.1. For the installation method of openssl upgrade, please refer to: [FAQ - Incorrect OpenSSL Vesion](https://docs.emqx.io/en/broker/v4.3/faq/error.html#incorrect-openssl-version)

<hr /><em>This discussion was created from the release <a href='https://github.com/emqx/emqx/releases/tag/v4.3.8'>EMQ X 4.3.8 Released</a>.</em>

**Comment by 1995parham** [comment]
@Rory-Z do we have the updated chart version for this release?

**Reply by Rory-Z** [reply]
Yes, we have
